website/identifiers/clients/base.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+
+class AbstractIdentifierClient(object):
+
+    def build_metadata(self, object):
```

Comment:
`object` is a reserved word in Python. How about `obj`?

Reply:
Also, it looks like `build_metadata` is only used internally by `create_identifier` and has no public usages. If that's the case, then this doesn't need to be part of the interface.